backend/badges/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```

Comment:
There is no need for exhaustive testing for this feature but a few simple tests can be added optionally.

Reply:
I'll get around to it if I'm done with all the feature implementations on time.